cmd/podman/common/netflags.go

```diff
@@ -164,11 +165,17 @@ func NetFlagsToNetOptions(cmd *cobra.Command) (*entities.NetOptions, error) {
 			return nil, err
 		}
 
+		parts := strings.SplitN(network, ":", 2)
```

Comment:
Recommendation: If length on this is 2, we should probably throw away `cniNets` below (options should only be given if you're not specifying more than 1 seems reasonable)

Reply:
fixed